### ISSUE TITLE
PDF reader: Allow window dragging from PDF reader toolbar

### ISSUE THREAD

**Issue by dstillman** [zotero/zotero#2713]
Is this hard due to the iframe?

https://forums.zotero.org/discussion/98540/not-enough-space-to-grab-window-on-mac

**Comment by dstillman** [comment]
Yeah, I figured that part at least should be possible. I guess better than nothing, though weird if the rest of the toolbar doesn't work. Do we think there's any sort of hacky way to do this from outside the iframe?

**Comment by mrtcode** [comment]
> Do we think there's any sort of hacky way to do this from outside the iframe?

Yes, place multiple boxes with `-moz-window-dragging: drag` on top of the iframe. Boxes have to cover only the areas between buttons in the toolbar.

But, I doubt if it's worth to implement that…
